test/util/testnode/rpc_client.go

```diff
@@ -91,6 +94,17 @@ func StartGRPCServer(logger log.Logger, app srvtypes.Application, appCfg *srvcon
 	}, nil
 }
 
+func GetGRPCClient(address string, interfaceRegistry types.InterfaceRegistry) (*grpc.ClientConn, error) {
```

Comment:
added this function to create a grpc client with the most up to date interfaceRegistry, the previous client that was being created did not yet have all types registered.